bucket/fastfetch.json

```
@@ -0,0 +1,24 @@
+{
+    "version": "1.8.2",
+    "description": "Like neofetch, but much faster because written in C",
```

```suggestion
    "description": "Fastfetch is a neofetch-like tool for fetching system information and displaying them in a pretty way, written in pure C.",
```

This is more descriptive